assimilation_code/modules/assimilation/algorithm_info_mod.f90

```diff
@@ -40,12 +40,13 @@ module algorithm_info_mod
 integer, parameter :: OBS_PARTICLE       = 4
 integer, parameter :: UNBOUNDED_RHF      = 8
 integer, parameter :: GAMMA_FILTER       = 11
-integer, parameter :: BOUNDED_NORMAL_RHF = 101 
+integer, parameter :: BOUNDED_NORMAL_RHF = 101
+integer, parameter :: KERNEL_QCEF        = 102
 
 public :: obs_error_info, probit_dist_info, obs_inc_info, &
           init_algorithm_info_mod, end_algorithm_info_mod, &
           EAKF, ENKF, BOUNDED_NORMAL_RHF, UNBOUNDED_RHF, &
-          GAMMA_FILTER, KERNEL, OBS_PARTICLE
+          GAMMA_FILTER, KERNEL, OBS_PARTICLE, KERNEL_QCEF
```

Comment:
@jlaucar note from the standup about KERNEL_QCEF  choice of name

KERNEL is in the existing code but not selectable as a filter kind with QCEFF (since v11+).

https://github.com/NCAR/DART/blob/c8ddfac035a39bbb24451ced3b0aa5410ac62494/assimilation_code/modules/assimilation/assim_tools_mod.f90#L1361

Reply:
I renamed this to `KDE_FILTER`